### DISCUSSION TITLE
[Discussion] About wiki commit 9fd0bc3eaefb1f879666b24b7852209c91e1c6c8 and the block

### DISCUSSION THREAD

**Discussion by MasterInQuestion** [MasterInQuestion/talk#1] [uBlock Origin]
Below texts need refactoring. Probably complete rewrite.
    But to summarize simply:
    I didn't mean what I'm accused of.

[[
<b id="discussioncomment-3208603">[ q1800 @ CE 2022-07-22 19:11:48 UTC:
[ https://github.com/uBlockOrigin/uBlock-issues/discussions/1958#discussioncomment-3208603](https://github.com/uBlockOrigin/uBlock-issues/discussions/1958#discussioncomment-3208603)</b>
    @gorhill or @gwarser:
    Do you mind reverting this Wiki edit or at least taking a look into it? (A person asked a question earlier in the discussions about this subject and was told the Wiki answer was very clear as it stood. Then the issue was locked.)
    [https://github.com/uBlockOrigin/uBlock-issues/wiki/_compare/9fd0bc3eaefb1f879666b24b7852209c91e1c6c8](https://github.com/uBlockOrigin/uBlock-issues/wiki/_compare/9fd0bc3eaefb1f879666b24b7852209c91e1c6c8)
    I don't feel this belongs in the wiki. Especially with it pointing to someone's comment and also putting "`[ Which? ]`" in the paragraph to instigate or troll a reaction. I don't know what this formatting code "`<&>...</&>`" is trying to accomplish either. <b>]</b>

\----

<b id="discussioncomment-3208769">[ gorhill @ CE 2022-07-22 19:54:46 UTC:
[ https://github.com/uBlockOrigin/uBlock-issues/discussions/1958#discussioncomment-3208769](https://github.com/uBlockOrigin/uBlock-issues/discussions/1958#discussioncomment-3208769)</b>
    Reverted, thanks for the heads up. The commit had nothing to do with the question asked. I have blocked that user to prevent further vandalism. <b>]</b>
]]
    My apologies for bringing the discussion to the less proper place...
    But <span>@</span>gorhill's block caused me unable to reply at the right place.
\
\
== Detailed reply (of above 2) ==

    [ Quote gorhill @ CE 2022-07-22 19:54:46 UTC:
[ https://github.com/uBlockOrigin/uBlock-issues/discussions/1958#discussioncomment-3208769](#discussioncomment-3208769)
    Reverted, ... The commit had nothing to do with the question asked. ]
<^>    Er, the commit indeed had nothing to do with the question asked [ [ https://github.com/uBlockOrigin/uBlock-issues/discussions/2183](https://github.com/uBlockOrigin/uBlock-issues/discussions/2183) ].
    As clearly indicated in the edit description:
[[
Added info about the specific-generic filter ("*##${Selector}") (and tagged ambiguity) and the "generichide" directive. Added example for ":style( ... )" operator.
]]
(not displayed in the ["wiki/_compare"](https://github.com/uBlockOrigin/uBlock-issues/wiki/_compare/9fd0bc3eaefb1f879666b24b7852209c91e1c6c8) page, only in ["_history"](https://github.com/uBlockOrigin/uBlock-issues/wiki/Static-filter-syntax/_history) (before revert) )

    [ Quote (previous):
    I have blocked that user to prevent further vandalism. ]
<^>    Vandalism?..
    I'd suggest looking at the rendered page:
    [https://github.com/uBlockOrigin/uBlock-issues/wiki/Static-filter-syntax/9fd0bc3eaefb1f879666b24b7852209c91e1c6c8](https://github.com/uBlockOrigin/uBlock-issues/wiki/Static-filter-syntax/9fd0bc3eaefb1f879666b24b7852209c91e1c6c8)
    .
    While I admit that the pre-rendered Markdown indeed somewhat looks like vandalism (without scrutiny), but that's Markdown's inherent problem.
    [ More on the topic: [https://github.com/orgs/community/discussions/20921](https://github.com/orgs/community/discussions/20921) ]
\
\
    [ Quote q1800 @ CE 2022-07-22 19:11:48 UTC:
[ https://github.com/uBlockOrigin/uBlock-issues/discussions/1958#discussioncomment-3208603](#discussioncomment-3208603)
    A person asked a question earlier in the discussions about this subject and was told the Wiki answer was very clear as it stood. Then the issue was locked. ]
<^>    That question is about a different subject (though referred the wiki) and for different purpose.
    And I sincerely do not quite understand why it was locked... As I (perhaps others) still have information to add.

    [ Quote (previous):
    I don't feel this belongs in the wiki. Especially with it pointing to someone's comment and also putting "`[ Which? ]`" in the paragraph to instigate or troll a reaction. ]
<^>    Similar practice is normal on Wikipedia. (reference inclusion (under fair use) and tagging questionable contents)

    [ Quote (previous):
    I don't know what this formatting code "`<&>...</&>`" is trying to accomplish either. ]
<^>    Without such marking how to tell which part the tag is to be applied?
\
\
== Off-Topic ==

    [ Quote gwarser @ CE 2022-07-25 05:37:39 UTC:
[ https://github.com/uBlockOrigin/uBlock-issues/discussions/2183#discussioncomment-3218989](https://github.com/uBlockOrigin/uBlock-issues/discussions/2183#discussioncomment-3218989)
    You cannot "comment" in filter list maintained by someone else and downloaded from remote location. ]
<^>    I realized the point soon after my further questioning.
    Though <span>@</span>mapx- rashly locked the discussion even sooner before I had the opportunity to update the post...

**Comment by ghost** [comment]
What is this unusual text formatting? I don't understand what is what.

**Reply by MasterInQuestion** [reply]
Which are you referring to?
    The one in my uBO wiki commit, or here?

    ----

    Assumed "both".

    "What is this unusual text formatting?"
<^>    My flower-gathering result from various existed practices.
    Plus some my own invention.

    "I don't understand what is what."
<^>    Indeed a problem, probably somewhat related with writing habit.
    But no superior solution seems possible.

    Partly related:
    https://github.com/orgs/community/discussions/68734#discussioncomment-7508369

    ----

    Things should look better with some simple standard bracket matching algorithm:
    E.g. [https://github<i>.</i>com/microsoft/vscode/issues/186288#issue-1776467472](https://github.com/microsoft/vscode/issues/186288#issue-1776467472)

    Thanks for your interest anyway.

**Comment by MasterInQuestion** [comment]
@garry-ut99, would you help reaching @gorhill?
    Who blocked me years ago for no good reason.

    I have several findings of uBO; and I believe the corporation shall benefit all.

**Reply by garry-ut99** [reply]
Unfortunately I can't help you reach him, because that creature banned me for no good reason as well, along with vandalising hundreds of my useful technical comments:
- https://github.com/garry-ut99/test/issues/1#issuecomment-2468932596

<hr>

> gorhill: https://github.com/uBlockOrigin/uBlock-issues/discussions/1958#discussioncomment-3208769 : I have blocked that user to prevent further vandalism.

In that case, at least one time in your live follow your own rules and block yourself to prevent further vandalism on your own part.

**Reply by MasterInQuestion** [reply]
Sort of Bugzilla Brilliance...
    "Multiple failures to follow our etiquette guidelines."?

**Comment by MasterInQuestion** [comment]
Is it Reddit Peculiarity... Or piteous moderation..?
<img src="https://github.com/MasterInQuestion/attach/raw/main/MasterInQuestion/talk/1/P-Mod.webp" alt="    https://github.com/MasterInQuestion/attach/raw/main/MasterInQuestion/talk/1/P-Mod.webp" />
